data/eos-shell-extension-prefs.desktop.in.in

```diff
@@ -1,6 +1,6 @@
 [Desktop Entry]
 Type=Application
-_Name=GNOME Shell Extension Preferences
+_Name=Extensions
```

Comment:
It's annoying that it's wrapped:

![image](https://cloud.githubusercontent.com/assets/507354/20759055/1215971c-b71c-11e6-94c9-a9aa1418b34c.png)